upcloud/resource_upcloud_objectstorage_test.go

```diff
@@ -349,16 +349,20 @@ func verifyObjectStorageExists(accessKey, secretKey, name string) resource.TestC
 
 func verifyObjectStorageDoesNotExist(accessKey, secretKey, name string) resource.TestCheckFunc {
 	return func(state *terraform.State) error {
-		time.Sleep(time.Second * 3)
-		exists, err := doesObjectStorageExists(state, accessKey, secretKey)
```

Comment:
can you delete the trailing function and all its dependancies

Reply:
Requires state *terraform.State to work and this seems to be good way to do it.